app/controllers/home_controller.rb

```diff
@@ -0,0 +1,11 @@
+class HomeController < ApplicationController
+
+  def all_media
+  	# x is the number of items that will show in the 'top' display
+  	x = 10
```

Comment:
Naming variables are hard! But I'd suggest renaming this to something other than a letter. Maybe 'top', 'top number', 'number_of_items' or something similar? "x" doesn't give you much info, hence the need for your comment.